packages/lesswrong/server/migrations/20250717T195519.make_auto_content_evalutions_nullable.ts

```diff
@@ -0,0 +1,12 @@
+export const up = async ({db}: MigrationContext) => {
+  await db.none(`ALTER TABLE "AutomatedContentEvaluations"
+    ALTER COLUMN "score" DROP NOT NULL,
+    ALTER COLUMN "sentenceScores" DROP NOT NULL,
+    ALTER COLUMN "aiChoice" DROP NOT NULL,
```

Comment:
This migration failed because the DB (both prod and dev) contains rows where this field is actually null